recipes/ixwebsocket/meta.yaml

```
@@ -0,0 +1,90 @@
+{% set name = "ixwebsocket" %}
+{% set version = "11.4.5" %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/machinezone/IXWebSocket/archive/v{{ version }}.tar.gz
+  sha256: c5fc225edec32bf7d583e55347ef2c9c4940d005c13ef5e848354a85602f5fd6
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - make
+  host:
+    - zlib
+    - openssl
+  run:
+
+outputs:
+  - name: libixwebsocket
+    build:
+      run_exports:
+        - {{ pin_subpackage('libixwebsocket') }}
```

Since the SOVERSION changes every patch version
```suggestion
        - {{ pin_subpackage('libixwebsocket', max_pin="x.x.x") }}
```